src/main/java/ch/njol/skript/expressions/ExprSortedList.java

```diff
@@ -38,7 +38,7 @@
 
 @Name("Sorted List")
 @Description("Sorts given list in natural order. All objects in list must be comparable; usually if you think you can compare it, it can be compared.")
-@Examples({"set {_list::*} to sorted {_list::*}"})
+@Examples({"set {_list::*} to sorted {_players::*}"})

```

Comment:
Both lists should use the same name, the example acts like a sort list **effect**.
```suggestion
@Examples({"set {_list::*} to sorted {_list::*}"})
```